linodecli/plugins/k8s-alpha.py

```diff
@@ -281,6 +299,11 @@ def dep_installed(command):
     except: 
         return False
 
+def terraform_version_supported(min_version, max_version):
+    # The Terraform version is of the format "Terraform v0.0.0"
+    version = check_output(['terraform','version']).split()[1].replace('v','',1)
```

Comment:
This breaks in python3.  I think this'll fix it for both 2 and 3:

```suggestion
    version = check_output(['terraform','version']).split()[1].decode().replace('v','',1)
```